pkg/tsdb/elasticsearch/healthcheck_test.go

```diff
@@ -35,35 +35,47 @@ func Test_Healthcheck_Timeout(t *testing.T) {
 	assert.Equal(t, "Elasticsearch data source is not healthy", res.Message)
 }
 
+func Test_Healthcheck_Error(t *testing.T) {
+	service := GetMockService(http.StatusBadGateway, "502 Bad Gateway")
+	res, _ := service.CheckHealth(context.Background(), &backend.CheckHealthRequest{
+		PluginContext: backend.PluginContext{},
+		Headers:       nil,
+	})
+	assert.Equal(t, backend.HealthStatusError, res.Status)
+	assert.Equal(t, "Elasticsearch data source is not healthy. Status: 502 Bad Gateway", res.Message)
```

Comment:
more or less a nit but do we ever define error strings as a const to reference them in multiple files? as it stands the test will fail if we update this messaging and the test would manually have to be updated to fix this 🤔

Reply:
We do sometimes define error variables (they can't be consts in go), but for this one I didn't bother because it's just a string explaining the failure to the user.